src/EvalExpressionAnalysis.cpp

```diff
@@ -0,0 +1,25 @@
+#include "EvalExpressionAnalysis.h"
+
+EvalExpressionAnalysis::EvalExpressionAnalysis(
```

Comment:
How about to move all the simple implementation into header files?

Reply:
Mixed feelings about this. On the one hand, it makes navigation difficult because implementation is split between `.cpp` and `.h` files. On the other hand, compiler will inline method definitions declared in `.h` files.